snuba/subscriptions/store.py

```diff
@@ -8,7 +9,32 @@
 from snuba.subscriptions.data import PartitionId, SubscriptionData
 
 
-class RedisSubscriptionDataStore:
+class SubscriptionDataStore(abc.ABC):
+    @abc.abstractmethod
+    def create(self, subscription_id: UUID, subscription: SubscriptionData) -> None:
```

Comment:
Kind of surprised and disappointed that mypy didn't catch this — the types here and in `delete` match `RedisSubscriptionDataStore`, but the argument names don't. I changed this in https://github.com/getsentry/snuba/pull/706#discussion_r368192750 so that it'd be easier to pull out into a generic later as a mapping that handled encoding/decoding values, which seems useful. We don't have a clear use case for it at the moment, so I don't really mind which approach we take, but it should be consistent. (Personally, I think `subscription_id` is a bit overloaded with `SubscriptionIdentifier` but I guess you could make a reasonable argument that `key` is overly generic as well.)